README.md

```diff
@@ -0,0 +1,20 @@
+# GPU-DEMO
+Uses deepspeech and tensorflow to recognize voice recording in english and translate using google translate to the desired language. Output of what was recognized in the recording and translation done to stdout.
+
+## Prerequisites
+* A wave file recorded in English using 16Khz and mono. Should be saved as 16bit PCM.
+* Python2
+* Python2-devel
+* Development tools and libraries (gcc, etc)
+* Python modules
+  * deepspeech or deepspeech-gpu
+  * googletrans
+  * jamspell
+  * numpy
+  * webrtcvad
+
+## Usage
+The translation language can be anything supported by [google translate](https://cloud.google.com/translate/docs/languages). Use the ISO-639-1 code.
+```
+$ python2 stream_with_sentences.py --lang <en|de|es|pl> --file </path/to/.wav> --models </path/to/models>
```

Comment:
This readme needs an update for your latest changes (--slang and --tlang).

For the source language you're offering English, French, and Russian for the spell checking part, but AFAIK the only pre-trained model offered from DeepSpeech is the English model (and that's what's being downloaded in the Dockerfile). Were you thinking of the possibility of users providing their own French or Russian language models for DeepSpeech? I think that could work. I was thinking, in that case, we should probably update the readme to briefly explain how ``--slang en|fr|ru`` could be used (have to provide your own models).